samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -16,7 +16,11 @@ Resources:
         Properties:
             CodeUri: src/HelloWorld/bin/Debug/netcoreapp2.0/publish
             Handler: HelloWorld::HelloWorld.Function::FunctionHandler
+            {%- if cookiecutter.runtime == 'dotnetcore2.0' %}
             Runtime: dotnetcore2.0
```

Comment:
Can runtime just be the value of `cookiecutter.runtime`?

Reply:
Done for explicit and valid runtime value